script/desktop/10.0/submitdiag/index.php

```diff
@@ -84,9 +86,9 @@
   echo "<result>$topic_id</result>";
 
   function CallDiscourse($path, $data, $method = 'GET', $username = '') {
-    $discourse_key = $_SERVER['api_keyman_com_discourse_key'];
-    $discourse_site = $_SERVER['api_keyman_com_discourse_site'];
-    $discourse_username = $_SERVER['api_keyman_com_discourse_username'];
+    $discourse_key = $env['api_keyman_com_discourse_key'];
+    $discourse_site = $env['api_keyman_com_discourse_site'];
+    $discourse_username = $env['api_keyman_com_discourse_username'];
```

Comment:
These need a `global $env;` to be able to read the global. Yeeeuch PHP

```suggestion
    global $env;
    $discourse_key = $env['api_keyman_com_discourse_key'];
    $discourse_site = $env['api_keyman_com_discourse_site'];
    $discourse_username = $env['api_keyman_com_discourse_username'];
```